en/docs/learn/analytics/managing-dashboard-permissions.md

```diff
@@ -12,6 +12,101 @@ A dashboard has the following three levels of permissions.
 
 ## Configuring Dashboard Permissions
 
+To support fine-grained access and permission control, WSO2 API Manager Analytics uses a set of scopes. Those scopes are assigned to each dashboard, mapping to different permission levels described above. The following table illustrates the default mapping between scopes and permission levels for each dashboard.
```

Comment:
WSO2 API Manager Analytics uses a set of scopes  - > WSO2 API Manager Analytics makes use of a set of scopes

Reply:
Are the above permission levels mapped to roles? If so, we can add the role in the table or simply point to the role mapping table in the user management?

Reply:
Not exactly. The permission levels are not directly mapped with the roles. A set of scopes can be assigned to permission levels, and since scopes are directly mapped with roles, the mapping between permission levels and roles can be derived. Also, that is customizable by assigning scopes to permission levels. Since this section focuses on that customization, didn;t added the table here. The following section "Permission level and user role mapping" contains the table which maps the default permission level to default user roles. Further, the permission levels are defined for the analytics only, we can't point the role mapping table for this purpose.

Reply:
> WSO2 API Manager Analytics uses a set of scopes - > WSO2 API Manager Analytics makes use of a set of scopes

Fixed with https://github.com/wso2/docs-apim/pull/1697/commits/ab12d89bb02874fd1eccd6b48d0a387a8ccd7ffa